apps/pwabuilder/tsconfig.json

```diff
@@ -19,7 +19,7 @@
       "isolatedModules": true,
       "resolveJsonModule": true,
       "types": [
-          "node"
+          "node", "vite/client"
```

Comment:
Why was this added?

Reply:
I am not sure

Reply:
@khmyznikov any info on this?

Reply:
Resolving as its not breaking the build in any way